### ISSUE TITLE
HTML in column title.

### ISSUE THREAD

**Issue by yangwen2** [akveo/ng2-smart-table#397]
I have a use case where the column header needs to have iconography.  Is it possible to have HTML content as column title?

**Comment by Anandamohinidasi** [comment]
+1 I also have the very same issue

**Comment by yangwen2** [comment]
I'm hacking it right now.  Assigned each column a unique css class.  Then injecting dom elements into the ng2-smart-table-title element in my component's AfterViewInit function.

**Comment by cgebe** [comment]
bump.

**Comment by PK1NONLY** [comment]
@yangwen2  did you get the solution?
if yes plz share

**Comment by yangwen2** [comment]
Yes like I said in the June 20 comment above.  I have a hack work around.

**Comment by samir-kamble** [comment]
Is there any other way than this??

**Comment by korifeich** [comment]
Here is the way I add icon non interactive to header using pure css:
```
.slash-outlined {
  .ng2-smart-title {
    display: inline-block;
    white-space: nowrap;

    &::before {
      display: inline-block;
      width: 20px;
      vertical-align: middle;
      margin-right: 5px;
      content: url(/assets/images/slash-outline.svg);
    }
  }
}
```
There result
![image](https://user-images.githubusercontent.com/3656411/194294075-51919163-93c8-4818-9fec-9fb6e963eab0.png)
